packages/yargs/lib/request.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const morgan = require('morgan');
+const split = require('split');
+
+const prettyMS = require('pretty-ms');
+const prettyBytes = require('pretty-bytes');
+const chalk = require('chalk');
+
+morgan.token('pretty-size', (req, res) => {
+  const size = res.get('content-length');
+  if (!size) return;
```

Comment:
What does morgan do with `undefined`? Does it just print nothing? Or convert it to a string and actually print `undefined`?

Reply:
It prints a `-` if it gets an `undefined`.